models/mrsegmentator/dockerfiles/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM mhubai/base:latest
+
+# Update Python, nnUNet needs Python >= 3.9
+#RUN apt-get update && apt-get install -y \
+#    python3.11 \
+#    python3-pip
+    
+# clone mhub implementation
+ARG MHUB_MODELS_REPO
+RUN buildutils/import_mhub_model.sh mrsegmentator ${MHUB_MODELS_REPO}
+
+# Install MRSegmentator
+RUN uv pip install -p /app/.venv mrsegmentator
```

Comment:
FYI; uv should use `.venv` by default.

Reply:
If you want to make sure there's no dependency conflict between mhubio and mrsegmentator you can use uv to create a custom venv and write a small cli wrapper running in that venv that can be called from within your Runner IO Module via self.subprocess. 

Example:
https://github.com/MHubAI/models/blob/main/models/pyradiomics/dockerfiles/Dockerfile#L3-L8